impl/src/main/java/com/okta/sdk/impl/resource/DefaultPasswordPolicyBuilder.java

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright 2017 Okta
```

Comment:
nit: this should be `2020 Okta, Inc.`
(I personally take the blame for not updating the header template)
@jeyadevvishnu don't worry about his one, we can do it if you don't get a chance.

Reply:
I will correct these with the upcoming commit. Thank you for pointing it out.

Reply:
I shall update the template shortly.